bw2io/strategies/ecospold2.py

```diff
@@ -24,7 +78,45 @@ def link_biosphere_by_flow_uuid(db, biosphere="biosphere3"):
 
 
 def remove_zero_amount_coproducts(db):
-    """Remove coproducts with zero production amounts from ``exchanges``"""
+    """
+    Remove coproducts with zero production amounts from the 'exchanges' list.
```

Comment:
Here you use a combination of both imperative and declarative.

Reply:
resolved